.github/workflows/docker_latest.yaml

```diff
@@ -0,0 +1,31 @@
+name: Docker
```

Comment:
I think we shouldn't just duplicate the build flow. If we change something in the future we need to remember to change it in both workflows. 

I'd either do something like this on release and rely on the fact that we separately build and push all GH tags to Docker.

```
    steps:
      - name: Login to Docker Hub
        uses: docker/login-action@v2
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_TOKEN }}
      - name: Retag and push as latest
        run: |
            IMAGE_NAME=scrolltech/l2geth
            RELEASE_TAG=${{ github.ref_name }} # should be the same as the release tag which we built the image before
            docker pull $IMAGE_NAME:$RELEASE_TAG
            docker tag $IMAGE_NAME:$RELEASE_TAG $IMAGE_NAME:latest
            docker push $IMAGE_NAME:latest
```

Or something like [this](https://blog.derlin.ch/github-actions-reusable-workflow-docker-images#heading-tags) is probably most elegant. 
If using the `docker/metadata-action@v3`, we could run this build workflow on tag and release (yeah the build for the release is wasted) and add multiple tags to each built. Eg for the `v5.4.2` release the following:
```
latest, scroll-v5.4.2, scroll-v5.4, scroll-v5
```
where latest is only updated on releases but the others according to their semver. But maybe out of scope of this PR.